static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -107,7 +108,7 @@ const getIndexSiteId = (): string => {
     } else {
         const site = config
             .get('page.pbIndexSites')
-            .find(s => s.bp === getBreakpointKey());
+            .find(s => s.bp === getBreakpointKey(), []);
```

Comment:
ah, put this on the wrong line... should have been `config.get('page.pbIndexSites', [])`

`config.get` takes an optional second parameter that will be used as the fallback if no matches on the first are found. Otherwise `config.get` returns undefined, and we probably do not want to call `.find` on `undefined` 😸